lunatrace/bsl/frontend/src/pages/project/builds/VulnerablePackageListItem/VulnerablePackageCardBody.tsx

```diff
@@ -141,7 +141,7 @@ const PackageDetails: React.FunctionComponent<PackageDetailsProps> = ({ pkg }) =
           <h5>
             {' '}
             <span className="darker">Language: </span>
-            {capitalizeFirstLetter(pkg.language)}
+            {toTitleCase(pkg.language)}
```

Comment:
I realized we can do this with CSS, it's just `.capitalize` in bootstrap i believe